src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx

```diff
@@ -26,39 +28,54 @@ const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
   crReviewed
 }) => {
   const faded = crReviewed != null && proposedSolution.approved === false;
+  const theme = useTheme();
 
   return (
-    <PageBlock title="" style={{ opacity: faded ? 0.5 : 1 }}>
-      {showDeleteButton && onDelete !== undefined ? (
-        <Button
-          color="error"
-          variant="outlined"
-          onClick={() => {
-            onDelete(proposedSolution);
-          }}
-          sx={{ position: 'absolute', right: 75 }}
-        >
-          <DeleteIcon />
-        </Button>
-      ) : null}
-      <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
-        <Grid item xs={7}>
+    <Grid
+      style={{ opacity: faded ? 0.5 : 1 }}
+      sx={{
+        minHeight: 300,
+        p: 2,
+        mb: 5,
+        bgcolor: theme.palette.background.paper,
```

Comment:
if you can I think making the background color slightly different (maybe the grey of the background in the popup) and then removing the border color will make this look better